FlipViewExtensions implementation (#387)
unoplatform/uno.toolkit.ui: doc/helpers/Selector-extensions.md

# SelectorExtensions Attached Properties

Provides an easy integration between `PipsPager` and `Selector` controls.

## Properties
Property|Type|Description
-|-|-
PipsPagerProperty|PipsPager| Backing property for the `PipsPager` that will interact with the `Selector` control.

> When the `SelectorExtensions.PipsPager` is set, the control will take care of updating the number of items and the `SelectedIndex` automatically.

Reviewer: ```suggestion
When the `SelectorExtensions.PipsPager` is set, the control will take care of updating the `NumberOfPages` and `SelectedIndex` properties automatically.
```

## Usage
```xml

> xmlns:utu="using:Uno.Toolkit.UI"

Reviewer: ```suggestion
xmlns:utu="using:Uno.Toolkit.UI"
xmlns:muxc="using:Microsoft.UI.Xaml.Controls"
```

...

<Grid>
    <FlipView
        utu:SelectorExtensions.PipsPager="{Binding ElementName=pipsPager}"
        Background="Blue">

>     <msui:PipsPager

Reviewer: ```suggestion
    <muxc:PipsPager
```
